src/skmultiflow/meta/online_smote_bagging.py

```diff
@@ -144,6 +143,9 @@ def partial_fit(self, X, y, classes=None, sample_weight=None):
         self
 
         """
+        if self.adwin_ensemble is None:
```

Comment:
Should not `ensemble` be used here, instead? (to cope with other ensemble models)

Reply:
Correct, fixed in 0d7aaeeb2841b9e5dbfe80fb821691ee2fbbd166